.github/workflows/main.yml

```diff
@@ -0,0 +1,145 @@
+name: Docker Image CI
+
+on:
+  push:
+
+env:
+  NB_SERVICES: ${{ secrets.NB_SERVICES }}
+  OPTIMIZER_ORTOOLS_VERSION: ${{ secrets.OPTIMIZER_ORTOOLS_VERSION }}
+  PROJECT: optimizer
+  REGISTRY: ${{ secrets.REGISTRY }}
+  VROOM_VERSION: ${{ secrets.VROOM_VERSION }}
+
+jobs:
+  test_basis:
+    runs-on: ubuntu-latest
+    env:
+      OPTIONS: "COV=false LOG_LEVEL=info SKIP_DICHO=true SKIP_REAL_CASES=true SKIP_SCHEDULING=true SKIP_SPLIT_CLUSTERING=true"
+    steps:
+      - uses: actions/checkout@v1
+      - name: Cache
+        id: cache
+        uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}
+      - name: Build image
+        run: ./.github/actions/build.sh
+        shell: bash
```

Comment:
Can't we create the image just once and launch the tests after? I mean that looks more like proper workflow unless it is complicated.

Reply:
We cannot create the image and pass it to another job. At every job, or we build it, or we download it.

Reply:
So there is no way to set a common "environment" for all jobs :/ weird. Thanks Alain!